spec/luassert_helper.lua

```diff
@@ -1,7 +1,20 @@
 local busted = require('busted')
 local assert = require('luassert.assert')
+local spy = require('luassert.spy')
+local stub = require('luassert.stub')
 local util = require ('luassert.util')
 local say = require('say')
+local tablex = require('pl.tablex')
+local deepcopy = tablex.deepcopy
+
+local original_stub_new = deepcopy(stub.new)
```

Comment:
stub.new is a function, there is no reason to copy it. just a reference will do.

```lua

stub_new = stub.new

function stub.new(...)
  return stub_new(...)
end
```